### PR TITLE
included script for finding fixed tickets. Issue #20464

### PR DESCRIPTION
**Pull request by poojadshende**

### CODE REVIEW

File: scripts/find_fixed_tickets.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```
Comment (johanandren) [inline scripts/find_fixed_tickets.sh]: A header that explains what it does without having to look this ticket up through git praise would be nice.

**Comment by johanandren** [comment]
LGTM (but a short docs-header explaining what it is would be nice)

**Comment by patriknw** [comment]
Refs #20464

**Comment by patriknw** [comment]
LGTM

**Comment by drewhk** [comment]
LGTM, although I won't be able to run this :(

**Comment by poojadshende** [comment]
@johanandren Added description header explaining what script is!

**Comment by johanandren** [comment]
Great, thanks!

**Comment by johanandren** [comment]
LGTM

**Comment by 2beaucoup** [comment]
No big deal and maybe the ticket description was a bit misleading but a quick check back before copying this over would have been nice.

**Pull request merged**